### ISSUE TITLE
Witch hut bounding box

### ISSUE THREAD

**Issue by pomid** [gniftygnome/skylark#5]
hello
the witch hut bounding box is being erase along with everything else
could use them huts bounding box for a witch farm

**Comment by gniftygnome** [comment]
Yes, since structure placement is suppressed, there would be no bounding boxes.  I'd have to implement my own version of structure placement and learn how to just submit the POI data ... probably.  This might happen, but it's likely to be quite a bit of work.

**Comment by pomid** [comment]
check this out mate
it's for a bit older version but works
https://github.com/jsorrell/CarpetSkyAdditions
